packages/lesswrong/components/comments/CommentsItem/CommentsItemMeta.tsx

```diff
@@ -272,11 +274,16 @@ export const CommentsItemMeta = ({
           {moderatorCommentAnnotation}
         </span>
       }
-      {!comment.debateResponse && !comment.rejected && <SmallSideVote
-        document={comment}
-        collection={Comments}
-        hideKarma={post?.hideCommentKarma}
-      />}
+
+      {!comment.debateResponse && !comment.rejected && allowVoting 
+        ? <SmallSideVote
+          document={comment}
+          collection={Comments}
+          hideKarma={post?.hideCommentKarma}
+        /> 
+        : <PostsItem2MetaInfo className="LWPostsItem-karma">
```

Comment:
I don't think we should be adding this className manually. It looks like all it's doing is adding some left margin, which we could just do with a new class in the `styles` object at the top of this file (the relevant className will then be available on the `classes` object that gets passed in as a prop to this component).

Reply:
done